docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+-- | -- | -- | -- | -- | -- | -- | -- | --
+t_5298036_b_77503_tn_0 | area_713 | 1 | 06:30:00 | 20:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298036_b_77503_tn_0 | area_714 | 2 | 06:30:00 | 20:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298041_b_77503_tn_0 | area_713 | 1 | 09:00:00 | 19:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298041_b_77503_tn_0 | area_714 | 2 | 09:00:00 | 19:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298046_b_77503_tn_0 | area_714 | 1 | 06:30:00 | 20:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298046_b_77503_tn_0 | area_713 | 2 | 06:30:00 | 20:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298051_b_77503_tn_0 | area_714 | 1 | 09:00:00 | 19:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298051_b_77503_tn_0 | area_713 | 2 | 09:00:00 | 19:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+
+## On-demand services where riders must board and alight at specific locations
+
+In certain demand-responsive services, riders are unable to specify boarding and alighting at any location within a zone. Instead, riders can only book to board and alight at specific designated stops(collection points/ virtual stops). An example of this is the [RufBus service](https://uvg-online.com/rufbus-angermuende/) in Angermünde and Gartz, Germany:
+
+### Define trips
+
+Route 476 offers on-demand services between each stop in the Angermünde region, including two trips(services): one for weekdays and the other for weekends. 
+
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id 
+-- | -- | -- 
+476 | on_demand_weekdays | 476_weekdays 
+476 | on_demand_weekends | 476_weekends 
+
+### Define location groups
+
+As riders can book services between each stop, to avoid defining all stop-to-stop combinations in stop_times.txt, the appropriate approach is to define these stops as a location group using location_groups.txt and location_group_stops.txt.
+
+[**location_groups.txt**](../../reference/#location_groupstxt)
+
+location_group_id | location_group_name 
+-- | -- 
+476_stops | durch den RufBus 476 bedientes Gebiet im Raum Angermünde
+
+[**location_group_stops.txt**](../../reference/#location_group_stopstxt)
+
+location_group_id | stop_id 
+-- | -- 
+476_stops | de:12073:900340004::1
+476_stops | de:12073:900340004::2
+476_stops | de:12073:900340004::3
+476_stops | de:12073:900340004::4
+476_stops | de:12073:900340100::1
+476_stops | de:12073:900340100::2
+476_stops | ...
+
+### Define booking rules
+
+- The 476 route service requires booking at least one hour in advance. Using `booking_type = 1` indicates that the service requires up to same-day booking with advance notice. The `prior_notice_duration_min = 60` indicates a requirement for booking at least 60 minutes in advance.
```

Comment:
Again I'd remove the bullet points for consistency, this is not really a list.

Reply:
Updated format